inc/saiacl.h

```diff
@@ -1579,10 +1579,19 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_FIELD_DST_PREFIX_META = SAI_ACL_TABLE_ATTR_FIELD_START + 0x161,
 
+    /**
+     * @brief Nexthop DST User metadata
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default false
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_NEXT_HOP_DST_USER_META = SAI_ACL_TABLE_ATTR_FIELD_START + 0x162,
```

Comment:
Do we need to include DST in the attribute name? I understand that corresponding neighbor attribute also has DST in the attribute name but port, vlan and acl attributes don't. In my opinion, DST in the attribute name is required for FDB and ROUTE objects only.